src/test-run/commands/validations/argument.js

```diff
@@ -28,10 +28,9 @@ import { assertUrl } from '../../../api/test-page-url';
 // Validators
 export var integerArgument         = createIntegerValidator(ActionIntegerArgumentError);
 export var positiveIntegerArgument = createPositiveIntegerValidator(ActionPositiveIntegerArgumentError);
-export var booleanArgument         = createBooleanValidator(ActionBooleanArgumentError);
+export var booleanArgument         = createBooleanValidator(ActionBooleanOptionError);
```

Comment:
[`ActionBooleanOptionError`](https://github.com/DevExpress/testcafe/blob/27fb1e7195db32bb2acf9721b1d23330c2458cb4/src/errors/test-run/templates.js#L49) has a text for options, not for argument